interface/patient_file/summary/immunizations.php

```diff
@@ -108,6 +111,10 @@
         $result = sqlQuery($sql, array($_GET['id']));
 
         $administered_date = new DateTime($result['administered_date']);
+        $uuid = null;
+        if (isset($result['uuid']) && UuidRegistry::isValidUUID($result['uuid'])) {
```

Comment:
oh, I see that you are using isValidUUID function for both binary and string validation. i would probably make those 2 separate functions (the only real difference is that check for `'\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0'` but since we are making these new helper functions, good idea to make it clear).
And come to think of it, this isn't really checking if valid, but more checking if empty. so would use names of function to be similar to something like:
isEmptyStringUUID
isEmptyBinaryUUID

Reply:
Thanks, that would be better :)